content/post/perfect-analytics-org-structure.md

```diff
@@ -0,0 +1,54 @@
+---
+author: Felix Puetsch
+title: "The perfect Analytics Org Structure"
+publishDate: "2018-08-19"
+draft: true
+tags: 
+ - analytics
+ - org structure
+ - org chart
+ - governance
+ - org
+ - teams
+---
+
+
+Have we left the Oil Age and entered the Data Age? Some would argue so. Undoubtedly, there is more data available than ever, and we are just uncovering the tip on what's possible with it. Most companies have embraced this and are employing often multiple people in an Analytics function.
+
+But what's the best way to set your org structure up for success?
+
+The way different Analysts' reporting lines are set up falls into four distinct archetypes.
+
+## Scattered
+
+In a scattered Analytics function, different departments or units of the organization have employees that have various degrees of Analytical skills. Often, those are not formally part of their job title, but just happen to make them (or their teams) successful at their job. This might be the sales manager who happens to be able to run sophisticated and accurate forecast models, or the product manager who employs statistical analyses to spot anomaly trends in the performance of his products.
+
+While this set up can work well for a long time, you usually see it in companies who haven't realized as a whole the vlaue that insights from data bring. The lack of alignment and knowledge exchange between the individuals performing Analytics limits the value they can provide.
```

Comment:
It could be just me, but I think `companies who` sounds more odd than `companies that`, since companies aren't people..? (without getting into Citizens United or the Fourteenth Amendment)